waspc/cli/Command/Build.hs

```diff
@@ -14,14 +14,28 @@ import CompileOptions (CompileOptions (..))
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
 import qualified Lib
-import StrongPath (Abs, Dir, Path, (</>))
+import StrongPath (Abs, Dir, Path, toFilePath, (</>))
+import System.Directory
+  ( doesDirectoryExist,
+    removeDirectoryRecursive,
+  )
 
 build :: Command ()
 build = do
   waspProjectDir <- findWaspProjectRootDirFromCwd
   let outDir =
         waspProjectDir </> Common.dotWaspDirInWaspProjectDir
           </> Common.buildDirInDotWaspDir
+      buildDirFilePath = toFilePath outDir
```

Comment:
To keep naming consistent here in this function, I would rename buildDirFilePath to outDirFilePath. Or, rename outDir to buildDir, that is also ok! Maybe even better.